packages/user-interface/src/utils/person-data.tsx

```diff
@@ -63,7 +63,7 @@ const getPostalCodeCityString = (
 };
 
 const getLocaleDateOfBirth = (
-  dateOfBirth: PersoonGeboorteDatum | null | undefined,
+  dateOfBirth: PersoonDatum | null | undefined,
```

Comment:
haha hoe kan dit in eens veranderd zijn?

Reply:
Blijkbaar is in de achterkant ooit het persoon type gewijzigd, en is daarna geen GraphQL codegen gedaan.